surveys/2021-annual-survey/questions.md

```diff
@@ -219,70 +219,213 @@ TODO ask about blockers for participation?
+> in one particular way.
+
+### Do you or your company use Rust at work?
+
+Select one:
+- Yes, I work with Rust full time
+- Yes, I work with Rust part time
+- No, but my company uses Rust 
+- No
+- Not sure 
+- Not applicable 
+
+> **REMOVED**
+>
+> This question was removed from the survey, because it was not sufficient in gathering the information we wanted.
 
-How reliant on Rust is your company?
+### For programmers - what languages, other than Rust, do you use at work?
+
+Free form:
 
-- It's the companies main language tech
-- It's a large part
-- It's a small part
-- We're experimenting
-- No use
+> **REMOVED**
+>
+> While this might be an interesting question, there's not too much that is directly actionable from it.
 
-And/or
+### Does your company plan to use Rust or evaluate Rust in the future?
 
-- Used in production across multiple projects
-- Used in production
-- Being actively developed
-- Experimented with
-- Being considered
+Select one:
+- Yes 
+- No
+- I don't know
 
-In what domains is Rust used at your company
+> **REMOVED**
+>
+> This data is better captured in other questions.
 
-- server networking, desktop application, mobile application, web application, embedded, etc.
+### How could we make Rust more appealing to your company?
+
+Free form:
+
+> **REMOVED**
+>
+> This question is too vague and unlikely to yield any interesting answers.
+
+### How may developers at your company use Rust at work?
+
+- 1
+- 2-5
+- 6-10
+- 11-25
+- 25-100
+- More than 100
+- I don't know
+
+> **REMOVED**
+>
+> This question fails to consider companies with different sizes. One company with 100,000 developers
+> where more than 100 use Rust is different than a company with 110 developers. We can gauge the importance
+> of Rust to companies through more direct questions
+
+### Approximately how many total developers does your company employ?
+
+- under 10
+- 11-49
+- 50-99
+- 100-500
+- 100-500
+- 500-1,000
+- 1,000-10,000
+- 10,000-100,000
+- Over 100,000
 
+> This question is not that interesting on its own, but it can be used as a sort of co-hort for understanding how answers 
+> change depending on the size of the development effort at a company.
+>
+> Previously this question used "employees" instead of "developers". It is more appropriate for us to ask about the amount
+> of developers at a company vs. the amount of people employed in total.
 
+### If you summed up the size of all Rust projects at work, how big would it be?
+
+Select one:
+- Less than 1,000 lines
+- 1,000 to 10,000 lines
+- 10,000 to 100,000 lines
+- More than 100,000 lines
+- I don't know
+
+> **REMOVED**
+>
+> While this might be interesting, there's nothing generally actionable from this. We are curious if Rust is playing a bigger
+> role in professional settings over time, and this is better served by other questions.
+
+### Is your company planning on hiring Rust developers in the next year?
+
+Select one:
+- Yes
+- No
+- I don't know
+
+> **REMOVED**
```

Comment:
I'm not sure about this one, at first I thought we should remove it, but on reflection I think it is quite a good question to assess hiring sentiment - although there is intrinsic uncertainty, it is easy to answer and forward looking.